src/renderer/shared/ui-kit/Select/Select.stories.tsx

```diff
@@ -37,34 +52,32 @@ export const RichContent: Story = {
     const [value, onChange] = useState('item_4');
 
     return (
-      <Box width="200px">
-        <Select {...args} placeholder="Select a fruit" value={value} onChange={onChange}>
-          <Select.Item value="item_1">
-            <Box direction="row" verticalAlign="center" gap={2}>
-              <Icon name="btc" size={12} className="shrink-0" />
-              <span className="truncate">Apple</span>
-            </Box>
-          </Select.Item>
-          <Select.Item value="item_2">
-            <Box direction="row" verticalAlign="center" gap={2}>
-              <Icon name="usd" size={12} className="shrink-0" />
-              <span className="truncate">Orange</span>
-            </Box>
-          </Select.Item>
-          <Select.Item value="item_3">
-            <Box direction="row" verticalAlign="center" gap={2}>
-              <Icon name="eur" size={12} className="shrink-0" />
-              <span className="truncate">Watermelon</span>
-            </Box>
-          </Select.Item>
-          <Select.Item value="item_4">
-            <Box direction="row" verticalAlign="center" gap={2}>
-              <Icon name="rub" size={12} className="shrink-0" />
-              <span className="truncate">Banana-nana-nana-nana-nana-nana</span>
-            </Box>
-          </Select.Item>
-        </Select>
-      </Box>
+      <Select {...args} placeholder="Select a fruit" value={value} onChange={onChange}>
+        <Select.Item value="item_1">
+          <Box direction="row" verticalAlign="center" gap={2}>
```

Comment:
If you remove this `<div className="h-full w-full truncate"/>` wrapper inside each Select.Item - it will omit usage of this box in every other place, would be great